### PR TITLE
CIV-2801 Application Closed

### PR DESCRIPTION
**Pull request by smitatalawadekar-hmcts**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [X] Screenshot attached for the Evidence of manual testing
- [X] Functional, Smoke and API tests have been run locally
- [X] Full gradle build and run tests with coverage have been completed (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-2801

### Change description ###
These changes are made to change application state once the claim is either dismissed or it proceeds offline

### Testing branch and instruction ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -8,6 +8,7 @@ def type = "java"
 def product = "civil"
 def component = "general-applications"
 def camundaBranch = "master"
+
```
Comment (karthick-mohanasundaram-HMCTS) [inline Jenkinsfile_CNP:11]: could we revert this change if it's been by accident? Thanks

**Review by deepthidoppalapudihmcts — DISMISSED** [review]

**Review by MMNycz — APPROVED** [review]

**Review by karthick-mohanasundaram-HMCTS — DISMISSED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Review by vasudevganesanhmcts — APPROVED** [review]

**Review by vasudevganesanhmcts — DISMISSED** [review]

**Review by vasudevganesanhmcts — APPROVED** [review]

**Comment by vasudevganesanhmcts** [comment]
merging as there were approvers for same code gone stale
